components/PostContent/PostContent.tsx

```diff
@@ -73,6 +74,14 @@ export default function PostContent( props: {
 								/>
 							);
 
+						case 'core/table':
+							return (
+								<Table
+									innerHTML={block.innerHTML || block.outerHTML}
```

Comment:
Is this OR in place to mitigate the bug in the block parsing, or does it serve another purpose?

Reply:
@chriszarate I'm not sure. I used the same structure used on `core/code`. However, on my tests, we're getting the `innerHTML`. Maybe we can remove this OR.